doc/book/adapters.md

```diff
@@ -150,10 +150,10 @@ All of the above options may be passed to the constructor by simply removing the
 The `Zend\Captcha\ReCaptcha` adapter uses [`Zend\Service\ReCaptcha\ReCaptcha`](https://github.com/zendframework/ZendService_ReCaptcha)
 to generate and validate CAPTCHAs.  It exposes the following methods:
 
-- `setPrivKey($key)` and `getPrivKey()` allow you to specify the private key to
+- `setSecretKey($key)` and `getSecretKey()` allow you to specify the secret key to
```

Comment:
Is this change within *this* component, or within the recaptcha component? If this component, we should likely retain the original methods, have them proxy to the new ones, and mark them deprecated.